docs/semgrep-app/editor.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: editor 
+append_help_link: true
+title: Editor 
+description: "Semgrep Editor is a powerful tool within Semgrep App to author rules and quickly apply these rules across an organization to enforce coding standards across an organization."
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Writing rules using Semgrep Editor
+
+
+![Screenshot of the Editor's splash screen](../img/editor-splashscreen.png)
+
+**Semgrep Editor (Beta)** is an advanced tool for **rule writing**. Write rules on a YAML editor, verify their performance through tests, and add them to your organization’s [Rule Board](https://semgrep.dev/docs/semgrep-app/rule-board/) to enforce code standards and increase code security.
+
+To quickly learn Semgrep patterns and syntax, explore the Editor’s library of rules from the **public [Rule Registry](https://semgrep.dev/r)**. Rules from the Registry can detect OWASP vulnerabilities, best practice violations, and security issues for a wide variety of languages and frameworks. Semgrep Editor enables you to **adapt these rules** for your own organization’s use by [forking](#jumpstart-rule-writing-using-existing-rules) them.
+
+The Editor is free to use on all tiers, but saving a rule for **private use** is a **Team/Enterprise tier feature**.
+
+## Accessing Semgrep Editor
+
+1. Sign in to your [Semgrep App account](https://semgrep.dev/login).
+2. Click **Editor (beta)** from the **App sidebar**. The **splash page** and **library pane** appear.
+3. Do any of the following steps:
+    1. To create a new rule, click on the **(+) plus sign** or **Create new rule** button.
+    2. To open any rule you’ve recently edited, select it from the **Recent** list.
+    3. To view a sample rule, select it from the **Examples** list. The rule renders within the Editor.
+    4. To start a tutorial or read the docs, select it from the **Learn** list. This navigates you away from the Editor.
+
+## Viewing rules
+
+![Screenshot of the Editor's main view composed of three panes and top menu](../img/editor-panes.png)
```

Comment:
Could I send you a different screenshot of this view? The Editor's splash page looks best if there are multiple rules in the "Recent" column (something like this)

<img width="1595" alt="Screen Shot 2022-02-08 at 9 04 44 AM" src="https://user-images.githubusercontent.com/19784302/153038117-280f11c3-5084-4c70-bb24-7aeb4dbffd69.png">

Reply:
I will use this photo you provided, thank you!